runtime/integration-tests/src/xcm/kusama/tests/transfers.rs

```diff
@@ -280,7 +280,7 @@ fn transfer_ausd_to_altair() {
 		// Sanity check the actual balance
 		assert_eq!(
 			OrmlTokens::free_balance(CurrencyId::AUSD, &BOB.into()),
-			16990730400000
+			16991917600000
```

Comment:
XCM fees changed ever so slightly

Reply:
Out of curiosity: How was this value chosen? I would expect the test failed and returned `16991917600000` as the expected value, originating from default ORML weights which supposedly were updated for `0.9.33`?